packages/editor-ui/src/stores/webhooks.store.ts

```diff
@@ -0,0 +1,66 @@
+import { STORES } from '@/constants';
+import type { IFakeDoor, INodeUi, IRootState } from '@/Interface';
+import type { IMenuItem } from 'n8n-design-system';
+import type { IWorkflowSettings } from 'n8n-workflow';
+import { defineStore } from 'pinia';
+import { useRootStore } from './n8nRoot.store';
+import { useNDVStore } from './ndv.store';
+import { useUIStore } from './ui.store';
+import { useUsersStore } from './users.store';
+import { useWorkflowsStore } from './workflows.store';
+
+export const useWebhooksStore = defineStore(STORES.WEBHOOKS, {
```

Comment:
sorry for the confusion.. I think in my head I want the hooks to be able to call any store, so that it's still useful for anyone still needing to write hooks. I see now this basic store came back which is not very useful